cs481-ekh/f21-welolo
# Updated build.sh and test.sh (#34)

Verification steps:
-  Pull branch and run ./build.sh and ./test.sh on your local machine
-  Both servers should start and shortly after the Cypress GUI will appear and allow you to run all tests

The github action currently just builds the file as pulling in the recent changes from main that require a .env file cause the build to fail. As we are unable to create the file in the repo without giving away all the secret keys, I had to take this section out of github actions and tests will have to be ran manually before making a PR.
